app/test/unit/git/worktree-test.ts

```diff
@@ -128,11 +128,14 @@ describe('git/worktree', () => {
     it('creates worktree at temporary path', async () => {
       const workTree = await createTemporaryWorkTree(repository, 'HEAD')
       const tmpDir = Os.tmpdir()
+      const directories = await FSE.readdir(realpathSync(tmpDir))
 
       expect(workTree.head).toBe(currentHeadSha)
       // we use realpathSync here because git and windows/macOS report different
       // paths even though they are the same folder
-      expect(realpathSync(workTree.path)).toStartWith(realpathSync(tmpDir))
+      const path = realpathSync(workTree.path).split(/\/|\\/)
+      const folder = path[path.length - 1]
```

Comment:
I think you can use `path.basename` for this:
```suggestion
      const folder = path.basename(workTree.path)
```

Reply:
Also probably worth writing a comment about why are we doing this instead of the `toStartWith` 🤔  Specially now that the test could start giving false  positives.

Reply:
Done - Great input!